v22.1/hashicorp-integration.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Hashicorp Vault Integration
+summary: Overview of uses cases for integrating CockroachDB with HashiCorp Vault
+toc: true
+docs_area: reference.third_party_support
+---
+
+This pages reviews the supported integrations between CockroachDB and [HashiCorp's Vault](https://www.vaultproject.io/).
+
+Vault offers tooling to extend CockroachDB's data security capabilities. Vault is an identity-based secrets and encryption management service, which can either be self-hosted, or accessed as a software as a service (SaaS) product through HashiCorp Cloud Platform (HCP).
+
+We officially support three integrations between Vault and CockroachDB:
+
+- [Using Vault's Key Management Secrets (KMS) Engine to manage & distribute encryption keys to AWS or GCP KMS for {{ site.data.products.dedicated }}'s customer-managed encryption key (CMEK) functionality.](#using-vaults-kms-secrets-engine-to-manage-a-cockroachdb-dedicated-clusters-customer-managed-encryption-key)
+- [Using Vault's Transit Secrets Engine to generate the store key for encryption at rest for a {{ site.data.products.core }} cluster.](#using-vaults-transit-secrets-engine-to-manage-a-cockroachdb-self-hosted-clusters-encryption-at-rest-store-key)
+- [Using Vault's PKI Secrets Engine to manage the server and client certificates for a {{ site.data.products.core }} cluster.](#using-vaults-pki-secrets-engine-to-manage-a-cockroachdb-self-hosted-clusters-server-and-client-certificates)
+
+In addition, because of CockroachDB's PostgreSQL compatibility, customers can also [use Vault's  PostgreSQL Database Secrets Engine to manage CockroachDB SQL user credentials]((#using-vaults-postgresql-database-secrets-engine-to-manage-cockroachdb-sql-users-and-their-credentials).
```

Comment:
```suggestion
Because of CockroachDB's PostgreSQL compatibility, customers can also [use Vault's  PostgreSQL Database Secrets Engine to manage CockroachDB SQL user credentials]((#using-vaults-postgresql-database-secrets-engine-to-manage-cockroachdb-sql-users-and-their-credentials).
```